openassessment/xblock/static/js/src/adg/lib/table_builder.js

```diff
@@ -358,6 +371,19 @@ $.fn.initORATableCheckbox = function (checkboxClass, disabled) {
             if (disabled) {
                 checkbox.prop('disabled', true);
             }
+
+            if($.browser.safari) {
+                checkbox.click(
+                    prependCaption
+                );
+
+                function prependCaption() {
+                    const table = _this.find('table');
+                    const caption = _this.find('caption');
+                    caption.detach();
+                    table.prepend(caption);
+                }
+            }
```

Comment:
Is there a way we might resuse the same code in two places?

Reply:
+1